docs/cookbook/configuration.rst

```diff
@@ -201,6 +201,17 @@ You can also set your tests to stop on failure by setting ``stop_on_failure``:
 
     stop_on_failure: true
 
+Moreover you can turn on verbosity by setting ``verbose``:
+
+.. code-block:: yaml
+
+    verbose: true
```

Comment:
I'd like to reserve the option to use the higher verbosity levels for something in future rather than just a boolean

Reply:
So you mean that accepted value should be `vvv` (or `3` that's debug value)? Is not counterintuitive? I mean, if I use `vvv` or whatever I would expect that also other levels are handled so I would try with `v` and `vv`.
WDYT?

Reply:
Yeah boolean's probably fine for now then